src/components/MeasureDisplay.js

```diff
@@ -50,3 +37,50 @@ export default class MeasureDisplay extends Component {
     );
   }
 }
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    onFetchMeasures: () => {
+      dispatch(fetchMeasuresIfNeeded());
+    }
+  };
+};
+
+function flattenMeasures(measures) {
+  let uniqueMeasures = [];
+  measures.forEach((m) => {
+    var reducedMeasure = {cmsId: m.cmsId, name: m.name, category: m.category};
+    if (! uniqueMeasures.map((i) => { return i.cmsId; }).includes(reducedMeasure.cmsId)) {
```

Comment:
Same comment here about the fat arrow, you can skip the curly braces and the return in `map`:

```js
uniqueMeasures.map((i) => i.cmsId)
```